### ISSUE TITLE
Answer this - take out choice of choosing a text or adding a question - have ONLY add a question

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#3589]
Unanimous decision: Take out the junction of choosing an existing text or adding a new question!

- [ ] Add the question box to the Content tab, with text “Written questions appear here” in it. This will help people understand that they can style the text, when they are in the TEXT panel in Step 2.
- [ ] If there is no kebab menu on the question box, then we must make sure that it is on the topmost layer. 
- [ ] Where the junction screen was, leave in only the **+ Add a question** part, and clicking it will open up the first question card.
- [ ] In the Interaction step, please add this explanation to the top of the tab: "Add and edit questions here"
- [ ] Jiggling help: "You can use text or audio to ask your students to 'Answer This'. The font, color and size of your questions, are editable in the text tab of step 2." 
- [ ] Add alert when trying to click or edit on canvas: “Add your question here” appears on left hand panel.
- [ ] Limit each question title to 2 lines.
- [ ] Enlarge the question field to at least 3 lines.
- [ ] When you add a new question, instead of the empty field, can there be default placehold text "New question', which will disappear as you start typing, or if there is no written text, audio only, it will not show in the preview or player?

**Comment by ChaykeeMor** [comment]
I know you're still in middle of this so please ignore if this is already on your radar.

- [ ] Default text MUST ONLY be editable in style..User cannot change the text that's there. Text only changes in question tab when inputting a question in left-hand panel.
- [ ] For correct answer tab - when dropping down to 'selected'.. either we have the first trace pre-selected so user sees one trace selected, and the audio options in panel... OR we have the 'select' graphic with the words "Select a trace to add feedback". Certainly cannot have the 'trace' graphic here instructing user to add new traces.
- [ ] For correct answer tab - when dropping down to 'all correct'.. Canvas should not look like everything is active. Canvas should indicate all correct traces as selected.

**Comment by johnnynotsolucky** [comment]
@ChaykeeMor RE 

> Default text MUST ONLY be editable in style

Do you mean teachers can only change the style of the text sticker, and not change the default text? If so, that might be slightly challenging - Right now, to change the style the sticker must be in edit mode which already allows them to change the text. I think this would mean some refactoring of the text sticker component itself. Could we have this as a separate task that doesn't block the Answer This work?

**Comment by corinnewo** [comment]
Editing default text is irrelevant as the default text will never be seen when playing. 

Sent from my iPhone

Corinne Ossendryver
CPO, Jewish Interactive 
(+27) 079-8865326

**Comment by ChaykeeMor** [comment]
The problem with being able to change text is precisely the issue we struggled with in the beginning of this entire flow. 1. We are trying to convey to the user that this is an automatically generated text-box based on your questions input in the left panel. 2. We don't want to confuse the tasks of designing your slide and entering your questions (why we mistakenly created a junction) 3. Having a text-box on your canvas that is framed in a different color is not enough of an indicator that this text-box is not a text-box and will disappear from your canvas. It needs very defined characteristics to understand why and how it functions differently.

**Comment by corinnewo** [comment]
I agree with Tyrone. Let's make this a separate issue that does not hold up this series of tasks. The default text states that the questions will go here. If people edit it, there is no actual effect. But the Jiggling text should help them understand that the orange box is a placeholder for the questions that they will define in the interaction stage.

**Comment by ChaykeeMor** [comment]
Ok. No problem..made a new issue. So for the first roll out.. 
What happens in the next step if a user DOES edit that box in the 'content' step? 

1. Input text returns to default text? and then no longer able to edit in the canvas? [get's that pop up in panel] 
2. Input text becomes the first question and skips the add a question step?? and if they used the question box as a title...they will somehow understand what they did wrong?
3. Input text becomes a title text and they will have to add a new question box?

**Comment by corinnewo** [comment]
I don't think any of those options are viable! Really confusing. A question box should remain a question box and not be changed into a title or anything else! If you try and edit the text, we should have an alert warning you that changing the text will not have any effect, since it is a placeholder for the questions defined in the interaction.

**Comment by johnnynotsolucky** [comment]
@ChaykeeMor for https://github.com/ji-devs/ji-cloud/issues/3589#issuecomment-1409881436 

> Canvas should indicate all correct traces as selected.

That would mean a refactor of the traces component to allow for more than one trace to be selected at a time, which could become quite a large bit of work. For now, I've made it so that it shows all the traces.

**Comment by ChaykeeMor** [comment]
@johnnynotsolucky As long as it's clear to the user that the same audio will play when gamer clicks on every correct answer...It does not have to look "selected" per se.
